src/main/java/tech/ypsilon/bbbot/discord/SlashCommandController.java

```diff
@@ -34,48 +36,53 @@
     public static final String BUTTON_PREFIX = "button";
     public static final String SELECT_MENU_PREFIX = "select";
 
-    private static SlashCommandManager instance;
+    private static SlashCommandController instance;
 
     private final Map<String, SlashCommand> commandMap;
 
     /**
-     * Creates a new {@link SlashCommandManager} and registers all {@link SlashCommand}s
-     *
-     * @param jda the {@link JDA} to register the commands on
+     * Creates a new {@link SlashCommandController} and registers all {@link SlashCommand}s
      */
-    public SlashCommandManager(JDA jda) {
+    public SlashCommandController(ButterBrot parent) {
+        super(parent);
         commandMap = new HashMap<>();
```

Comment:
I think you are missing `instance = this;`

Reply:
see c48b130